Mathlib/Algebra/Lie/LieTheorem.lean

```diff
@@ -0,0 +1,558 @@
+/-
+Copyright (c) 2024 Lucas Whitfield. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Lucas Whitfield
+-/
+import Mathlib.Algebra.Lie.Solvable
+import Mathlib.Algebra.Lie.Weights.Linear
+
+/-!
+# Lie's theorem for Solvable Lie algebras.
+-/
```

Comment:
Can you be more specific: such as, what is the precise statement you're formalising?

I presume your file includes other definitions or noteworthy lemmas on the way; can you mention these here as well? (In case you haven't read them, here's the documentation guide: https://leanprover-community.github.io/contribute/doc.html